tools/make/mixin.mk

```diff
@@ -0,0 +1,32 @@
+# Makefile.mixin adds mixin-specific targets.
+
+PARENT_MAKEFILE := $(firstword $(MAKEFILE_LIST))
+
+MIXIN_PATH       ?= operations/agent-flow-mixin
+MIXIN_OUT_PATH   ?= operations/agent-flow-mixin-compiled
```

Comment:
My initial use case was to mention the `make` command in the clustering docs PR so that people could easily reuse our dashboards and alerts. TBH I hadn't thought about including this as a release asset.

So you think this should be invoked among the other commands in `make dist`? I think it makes sense; and once we've fixed linting errors we can also make it part of our Drone setup, much like other build artifacts.

Reply:
> So you think this should be invoked among the other commands in make dist? I think it makes sense; and once we've fixed linting errors we can also make it part of our Drone setup, much like other build artifacts.

Hm, maybe? At first I thought that was what Mimir was doing, but I see they have the same -compiled folder as you do here. 

I'll leave it up to you, having it as a release asset potentially makes the workflow of installing it easier.